src/rpc/mining.cpp

```diff
@@ -524,14 +522,8 @@ static UniValue getblocktemplate(const JSONRPCRequest& request)
     result.pushKV("mintime", (int64_t)pindexPrev->GetMedianTimePast()+1);
     result.pushKV("mutable", aMutable);
     result.pushKV("noncerange", "00000000ffffffff");
-    int64_t nSigOpLimit = MAX_BLOCK_SIGOPS_COST;
-    int64_t nSizeLimit = MAX_BLOCK_SERIALIZED_SIZE;
-    assert(nSigOpLimit % WITNESS_SCALE_FACTOR == 0);
-    nSigOpLimit /= WITNESS_SCALE_FACTOR;
-    assert(nSizeLimit % WITNESS_SCALE_FACTOR == 0);
-    nSizeLimit /= WITNESS_SCALE_FACTOR;
-    result.pushKV("sigoplimit", nSigOpLimit);
-    result.pushKV("sizelimit", nSizeLimit);
+    result.pushKV("sigoplimit", (int64_t)MAX_BLOCK_SIGOPS_COST);
+    result.pushKV("sizelimit", (int64_t)MAX_BLOCK_BASE_SIZE);
     result.pushKV("curtime", pblock->GetBlockTime());
     result.pushKV("proof", HexStr(pblock->proof));
     result.pushKV("height", (int64_t)(pindexPrev->nHeight+1));
```

Comment:
In `getblocktemplate` results description still remain `weight` and `weightlimit` fields.

Reply:
@Naviabheeman this is not fix yet.

Reply:
sorry, my mistake.